client/src/js/components/bhMultipleDebtorGroupSelect.js

```diff
@@ -0,0 +1,51 @@
+angular.module('bhima.components')
```

Comment:
Go ahead and make another issue.  This report should get in as soon as possible.